src/firebase/firebase.controller.spec.ts

```
@@ -0,0 +1,197 @@
+    const dto = {
+      token: 'test-token-123',
+    };
+    const expectedResponse = {
+      success: true,
+      messageId: 'test-message-id',
+    };
+
+    mockFirebaseService.sendNotification.mockResolvedValue(expectedResponse);
+
+    // Act
+    const request = require('supertest');
```

ESLint

[error] A `require()` style import is forbidden.

(@typescript-eslint/no-require-imports)